besca/st/_wrapper_funcs.py

```diff
@@ -261,18 +261,23 @@ def clr_normalize(adata, results_folder):
     return(adata)
 
 
-def highly_variable_genes(adata, batch_key=None):
+def highly_variable_genes(adata, batch_key=None, n_shared=2):
```

Comment:
I think this should be quickly documented. It was never done.Could be the good occasion

Reply:
Ok done